README.md

```diff
@@ -33,9 +33,17 @@ These are the steps that the script takes:
 2. Ensure that the shell script is executable. In Mac / Unix, run `chmod +x deploy.sh`.
 3. Run the script with `sh deploy.sh`. You can also double-click it in Finder / Explorer to start it.
 4. You'll now be guided through a set of questions.
- 
+
 I prefer to keep this script in the root of my projects directory. Each project directory is named as the plugin slug, as is the corresponding GitHub repo. To use, just call the script, enter the plugin slug, confirm or amend default suggestions, and sit back as the code is sent to SVN and git repos including tags. The commit messages here are hard-coded for consistency.
 
+## Extras
+
+You may define your own `.plugins_svnignore` file similar to an `svn propset svn:ignore .plugins_svnignore` command. This will remove any listed files and/or directories from committing to the Plugins Directory.
```

Comment:
Why is the filename `.plugins_svnignore`? What's the origin of that name?

Reply:
I made it up. I initially had `.svnignore` but added `plugins_` for Plugins Directory SVN.

Reply:
I probably should just go with my first choice,  `.svnignore`